rc/filetype/git.kak

```diff
@@ -43,6 +51,13 @@ add-highlighter shared/git-commit/comments/ fill comment
 add-highlighter shared/git-commit/comments/ regex "\b(?:(modified)|(deleted)|(new file)|(renamed|copied)):([^\n]*)$" 1:yellow 2:red 3:green 4:blue 5:magenta
 }
 
+provide-module git-ignore %{
+add-highlighter shared/git-ignore group
+add-highlighter shared/git-ignore/glob regex '(?<!\\)\*((?<!\\)\*)?|(?<!\\)\?' 0:operator
```

Comment:
Nice idea. Some comments:

two backslashes followed by an asterisk means it's still a glob. To correct this we can allow a
`(?:\\\\)*` prefix (like sh.kak and others).

Also `\*((?<!\\)\*)?` can be simplified to `\*\*?`

```suggestion
+add-highlighter shared/git-ignore/glob regex '(?<!\\)(?:\\\\)*\K(\*\*?|\?)' 0:operator
```

I think we can also get away with adding character classes in this line:

```suggestion
+add-highlighter shared/git-ignore/glob regex '(?<!\\)(?:\\\\)*\K(\*\*?|\?|\[.*?(?<!\\)(?:\\\\)*\])' 0:operator
```

Reply:
This was a great suggestion. I completely forgot about the double backslash. I tested the new regex with character class support and it works just as expected so I have committed it.